src/DataExporter.ts

```diff
@@ -151,32 +157,34 @@ export class DataExporter {
     let ids: Id64Array = [];
 
     if (writeHeaders) {
-      const header: string[] = (options.calculateMassProperties) ? ["total_count", "volume", "volume_count", "area", "area_count", "length", "length_count"] : [];
-      const outHeader = this.makeHeader(header, statement, options.dropIdColumnFromResult ? options.idColumn : -1);
+      const header: string[] = (options.calculateMassProperties) ? ["volume", "volume_si", "area", "area_si", "length", "length_si"] : [];
+      const outHeader = this.makeHeader(header, statement, options.calculateMassProperties ? options.idColumn : -1);
       writeStream.write(`${outHeader}\n`);
     }
 
     let rowCount = 0;
     while (DbResult.BE_SQLITE_ROW === statement.step()) {
-      const stringifiedRow = this.rowToString(statement, options.dropIdColumnFromResult ? options.idColumn : -1);
+      const stringifiedRow = this.rowToString(statement, options.calculateMassProperties ? options.idColumn : -1);
       if (options.calculateMassProperties === true) {
         if (options.idColumnIsJsonArray === true) {
           ids = JSON.parse(statement.getValue(options.idColumn).getString()) as Id64Array;
         } else {
           ids = [statement.getValue(options.idColumn).getId()];
         }
+        // Uncomment to find the row in the query results.  Helpful if you have a crash and want to restart the calculations part way through a query.
+        // console.log(`Calculating mass properties for row ${rowCount} for ${ids.length} elements.`);
         const result = await this.calculateMassProps(ids);
-        writeStream.write(`${result.totalCount};${result.volume};${result.volumeCount};${result.area};${result.areaCount};${result.length};${result.lengthCount};${stringifiedRow}\n`);
+        writeStream.write(`${result.volume};${result.volumeCount / result.totalCount};${result.area};${result.areaCount / result.totalCount};${result.length};${result.lengthCount / result.totalCount};${stringifiedRow}\n`);
```

Comment:
switch back to the old output to correspond to switching back to the old headers